.rubocop.yml

```diff
@@ -49,7 +49,8 @@ Layout/LineLength:
 Style/BlockDelimiters:
   Exclude:
     - !ruby/regexp /.*_spec\.rb$/
-
+Style/HashSyntax:
+  Enabled: false
```

Comment:
I'm not a fan of [this cop](https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax) because it makes reading code more complicated: https://batsov.com/articles/2022/01/20/bad-ruby-hash-value-omission/